kolibri/plugins/learn/assets/src/views/ContentCard/index.vue

```diff
@@ -27,6 +27,7 @@
       </p>
       <div class="footer">
         <CoachContentLabel
+          v-if="!isLearner"
```

Comment:
Does this catch unauthenticated users as well?

Reply:
Thanks @jonboiser I just pushed a commit addressing that.